xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -276,49 +277,51 @@ ASS_Image* CDVDSubtitlesLibass::RenderImage(double pts,
     ApplyStyle(subStyle, opts);
   }
 
-  double sar = static_cast<double>(opts.sourceWidth) / static_cast<double>(opts.sourceHeight);
-  double dar = static_cast<double>(opts.videoWidth) / static_cast<double>(opts.videoHeight);
-  ass_set_pixel_aspect(m_renderer, dar / sar);
+  float par = (opts.m_par - 2.0f) * -1;
```

Comment:
Comment explaining this rather obscure calculation would be great :)

Reply:
yes i will ask to lord darth vader